src/Administration/Controller/AdministrationController.php

```diff
@@ -109,6 +112,19 @@ public function index(Request $request, Context $context): Response
         ]);
```

Comment:
Just saw the above endpoint. Couldn't we add the login config for the frontend (buttons) there instead of doing an additional request to fetch that data?